src/main/java/org/broadinstitute/hellbender/utils/dragstr/STRTableFile.java

```diff
@@ -0,0 +1,126 @@
+package org.broadinstitute.hellbender.utils.dragstr;
+
+import com.google.common.io.Files;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMSequenceDictionaryCodec;
+import htsjdk.samtools.util.BufferedLineReader;
+import htsjdk.samtools.util.LineReader;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.output.NullWriter;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.dragstr.DragstrLocus;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.ZipUtils;
+import org.broadinstitute.hellbender.tools.dragstr.STRDecimationTable;
+
+import java.io.*;
+import java.util.stream.Stream;
+
+/**
+ * Class to create and access STR table file contents.
+ */
+public final class STRTableFile implements AutoCloseable {
```

Comment:
As i have said elsewhere, i think this can become a feature reader which would open these sites files to be used for any number of other applications outside of the STR code.

Reply:
Ok. Probably a ticket.